tests/integration/conformance/main_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in conformance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conformance
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"istio.io/istio/pkg/test/conformance"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/istio"
+)
+
+func loadCases() ([]*conformance.Test, error) {
+	wd, err := os.Getwd()
```

Comment:
Doesn't conformance.Load accept relative paths? If not - filepath.Abs seems like a more succint version of what you're doing